frontend/admin/src/js/components/apiManagedTable/basicTableHelpers.tsx

```diff
@@ -11,11 +19,93 @@ export type ColumnsOf<T extends Record<string, unknown>> = Array<Column<T>>;
 // Information about a column in a table
 export type Column<T> = {
   label: string;
+  header?: React.ReactNode;
   id: IdType<T>;
   accessor: (object: T) => React.ReactNode;
   sortColumnName?: string; // API requires exact DB column name for sorting, not the name of the field in the API 😢
 };
 
+// This defines the row selection column
+export function rowSelectionColumn<T extends RecordWithId>(
+  intl: IntlShape,
+  selectedRows: T[],
+  pageSize: number,
+  labelAccessor: (item: T) => string,
+  onRowSelectionChange: (e: RowSelectedEvent<T>) => void,
+): Column<T> {
+  return {
+    label: intl.formatMessage({
+      defaultMessage: "Row Selection",
+      description:
+        "Label for the row-selection column in the tables column-selection modal.",
+    }),
+    header: (
+      <CheckButton
+        color="white"
+        checked={selectedRows.length === pageSize}
+        indeterminate={
+          selectedRows.length > 0 && selectedRows.length < pageSize
+        }
+        label={intl.formatMessage({
+          defaultMessage: "Select/Unselect all",
+          description: "Header label for the row-selection column in tables.",
+        })}
+        onToggle={() => {
+          onRowSelectionChange({ setSelected: selectedRows.length < pageSize });
+        }}
+      />
+    ),
+    accessor: (r) => {
+      const checked = selectedRows.includes(r);
+      return (
+        <CheckButton
+          checked={checked}
+          onToggle={() => {
+            onRowSelectionChange({
+              row: r,
+              setSelected: !checked,
+            });
+          }}
+          label={labelAccessor(r)}
+        />
+      );
+    }, // callback extracted to separate function to stabilize memoized component>
```

Comment:
Replace component> with component

Reply:
Fixed with [remove stray > in comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3279/commits/58f0ae645735d7e6bae60833d579f14c4aae2d77)